internal/settings/config_provider.go

```diff
@@ -264,30 +264,22 @@ func externalConfigToConfig(e ExternalConfig, dirPath string) (Config, error) {
 func getExcludePrefixesForDir(dirPath string) ([]string, error) {
 	filePath := filepath.Join(dirPath, DefaultConfigFilename)
 	if _, err := os.Stat(filePath); err != nil {
-		excludePrefixes := make([]string, 0, len(DefaultExcludePrefixes))
-		for _, defaultExcludePrefix := range DefaultExcludePrefixes {
-			excludePrefixes = append(excludePrefixes, filepath.Join(dirPath, defaultExcludePrefix))
-		}
-		return excludePrefixes, nil
+		return []string{}, nil
 	}
 	data, err := ioutil.ReadFile(filePath)
 	if err != nil {
 		return nil, err
 	}
 	s := struct {
-		ExcludePaths          []string `json:"excludes,omitempty" yaml:"excludes,omitempty"`
-		NoDefaultExcludePaths bool     `json:"no_default_excludes,omitempty" yaml:"no_default_excludes,omitempty"`
+		ExcludePaths []string `json:"excludes,omitempty" yaml:"excludes,omitempty"`
 	}{}
 	if err := yaml.Unmarshal(data, &s); err != nil {
 		return nil, err
 	}
-	return getExcludePrefixes(s.ExcludePaths, s.NoDefaultExcludePaths, dirPath)
+	return getExcludePrefixes(s.ExcludePaths, dirPath)
 }
 
-func getExcludePrefixes(excludes []string, noDefaultExcludes bool, dirPath string) ([]string, error) {
-	if !noDefaultExcludes {
-		excludes = append(DefaultExcludePrefixes, excludes...)
```

Comment:
Looks like we're removing `DefaultExcludePrefixes` entirely let's remove it's declaration from [here](https://sourcegraph.com/github.com/uber/prototool@50b7e7e874bde45db3da837f087cc7ba205434ec/-/blob/internal/settings/settings.go#L49:2).

I can't really see us re-introducing this, plus it's a minimal change if we want to add it back.